examples/ReactExample/components/login.js

```diff
@@ -0,0 +1,102 @@
+'use strict';
+
+import React from 'react';
+import { AsyncStorage, Text, TextInput, View } from 'react-native';
+import Realm from 'realm';
+import styles from './styles';
+import TodoApp from './todo-app';
+
+
+export default class LoginScreen extends React.Component {
+
+    constructor (props) {
+        super(props);
+        this.state = {login: 'user', password: '...'};
```

Comment:
Is the "..." password a placeholder? We should probably use the `placeholder` attribute of the `TextInput` element instead.